castle/cms/browser/controlpanel/status.py

```diff
@@ -0,0 +1,79 @@
+from Products.Five.browser import BrowserView
+import subprocess
+import redis
+import celery
+import celery.bin.base
+import celery.bin.celery
+import celery.platforms
+
+
+class StatusView(BrowserView):
+
+    def docsplit(self):
+        retval = self.DocsplitSearch()
```

Comment:
Since self.DocsplitSearch() is returning two values, for readability it might be better to assign the values to separate names, something like `success, msg = self.DocsplitSearch()` . And in this case you can also simply `return success` (or `return retval[0]` as it's named now) without the if statements, since we are returning True if it is True and False if it is False.